src/components/@shared/ButtonBuy/index.tsx

```diff
@@ -42,12 +43,14 @@ function getConsumeHelpText(
   assetType: string,
   isConsumable: boolean,
   isBalanceSufficient: boolean,
-  consumableFeedback: string
+  consumableFeedback: string,
+  isSupportedOceanNetwork: boolean,
+  web3: any
```

Comment:
we should use the proper type here instead of any
```suggestion
  web3: Web3
```